internal/rbd/migration.go

```diff
@@ -77,35 +77,41 @@ func parseMigrationVolID(vh string) (*migrationVolID, error) {
 // deleteMigratedVolume get rbd volume details from the migration volID
 // and delete the volume from the cluster, return err if there was an error on the process.
 func deleteMigratedVolume(ctx context.Context, parsedMigHandle *migrationVolID, cr *util.Credentials) error {
+	rv, err := genVolFromMigVolID(ctx, parsedMigHandle, cr)
+	if err != nil {
+		return err
+	}
+	defer rv.Destroy()
+	err = deleteImage(ctx, rv, cr)
+	if err != nil {
+		log.ErrorLog(ctx, "failed to delete rbd image: %s in rbdVol: %v, err: %v", rv.RbdImageName, rv, err)
```

Comment:
sorry if i was not clear what i meant was `log.ErrorLog(ctx, "failed to delete rbd image: %s, err: %v", rv, err)` as rv already have String method it will log `pool/radosnamespace/imagename`